windows-agent/internal/proservices/landscape/executor_test.go

```diff
@@ -223,6 +265,37 @@ func TestInstall(t *testing.T) {
 	}
 }
 
+func mockRootfsFileServer(t *testing.T, ctx context.Context) string {
```

Comment:
```suggestion
//nolint:revive // Context goes after testing.T
func mockRootfsFileServer(t *testing.T, ctx context.Context) string {
```
You'll see comments like those a lot above test helper functions.